extensions/amp-script/0.1/test/unit/test-amp-script.js

```diff
@@ -93,6 +94,19 @@ describes.fakeWin('AmpScript', {amp: {runtimeOn: false}}, env => {
     expect(service.checkSha384).to.be.called;
   });
 
+  it('should warn if there is zero width/height', () => {
+    const warnStub = env.sandbox.stub(user(), 'warn');
+    env.sandbox.stub(script, 'getLayoutBox').returns({height: 0, width: 0});
+    script.onLayoutMeasure();
+
+    expect(warnStub).calledWith(
+      'amp-script',
+      'Skipped initializing amp-script due to zero width and height.',
+      script.element
+    );
+    expect(warnStub).to.have.callCount(1);
+  });
```

Comment:
Worth adding a test for the positive use-case?

Poor pseudo-code here:
`env.sandbox.stub(script, 'getLayoutBox').returns({height: 10, width: 100});`
`expect(warnStub).to.have.callCount(0);`

Reply:
Done